### PR TITLE
Fixes unix timestamp for marketing preferences

### LINKED ISSUE #146: Fix Timestamp issue on giftaid
Timestamp currently is saving in this format `2019-02-25T12:30:03.000Z` and needs to be in unix format `1551107789`.

### PR DESCRIPTION
**Pull request by s-barrah**
Fixes #146 - unix timestamp for marketing preferences

### CODE REVIEW

File: src/pages/UpdateForm/UpdateForm.js [hunk 1]
```diff
@@ -170,8 +170,7 @@ class UpdateForm extends Component {
    * @return string
    */
   getTimestamp() {
-    const getTimeStamp = Math.round((new Date()).getTime() / 1000);
-    this.timestamp = new Date(getTimeStamp * 1000);
+    this.timestamp = Math.round((new Date()).getTime() / 1000);
```
Comment (Heleen-cr) [inline src/pages/UpdateForm/UpdateForm.js]: Same output, but why are we getting the unix timestamp here this way and in GiftaidForm.js via Date.now() ?

Comment (s-barrah) [reply src/pages/UpdateForm/UpdateForm.js]: I don't understand your question.

Comment (s-barrah) [reply src/pages/UpdateForm/UpdateForm.js]: `(new Date()).getTime()` and `Date.now()` practically do the same thing but one is cleaner and more precise.

Comment (Heleen-cr) [reply src/pages/UpdateForm/UpdateForm.js]: Should we then use the cleaner, more precise one in both cases?

Comment (s-barrah) [reply src/pages/UpdateForm/UpdateForm.js]: Yes we can. `Date.now()` performs better. `new Date().getTime()` is useful if you want a frozen `Date` instance or to convert a `YYYY-MM-DD` to a timestamp.

**Pull request merged**